daemons/controld/controld_based.c

```diff
@@ -46,6 +46,26 @@ do_cib_replaced(const char *event, xmlNode * msg)
     register_fsa_input(C_FSA_INTERNAL, I_ELECTION, NULL);
 }
 
+void
+disconnect_cib_manager(unsigned long long flags_to_clear)
```

Comment:
It's not a formal guideline, but it helps to use the "controld_" prefix for functions that will be used in more than one file, so that fact is obvious when reading the code.

Ideally, by looking at a function name, we'd know where it was defined: if it starts with pcmk_, it's from a library, and if it starts with a daemon name, it's defined somewhere in that daemon's code directory, and if it has no special prefix, it's defined in the local file. However we have a ton of old names that don't follow that model but can't be changed without breaking backward API compatibility.